kubevirt-disk-uploader-task-run.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: tekton.dev/v1beta1
+kind: TaskRun
+metadata:
+  name: kubevirt-disk-uploader-task-run
+spec:
+  taskRef:
+    name: kubevirt-disk-uploader-task
+  params:
+  - name: VM_NAME
+    value: example-vm-tekton
+  - name: VOLUME_NAME
+    value: datavolumedisk
+  - name: CONTAINER_DISK_NAME
+    value: example-vm-tekton-exported:latest
+  - name: ENABLE_VIRT_SYSPREP
+    value: false
+  serviceAccountName: kubevirt-disk-uploader-tekton
```

Comment:
remove this line. Keep the tekton to set default SA `pipeline`

Reply:
Done.